.github/workflows/setup-gcloud-it.yml

```diff
@@ -3,7 +3,7 @@ name: setup-gcloud Integration
 on:
   push:
     paths:
-    - 'setup-gcloud/**'
+    - './**'
     - '.github/workflows/setup-gcloud*'
```

Comment:
nit: Now that we have top leveled this, we can use
```yaml
  push:
    branches:
    - 'main'
  pull_request:
```
for parity with other actions test workflows
https://github.com/google-github-actions/deploy-cloudrun/blob/c8ab46dee1f5dad5d4ec6e92519bf90fa9318923/.github/workflows/deploy-cloudrun-it.yml#L3